website/splash_table.py

```diff
@@ -53,6 +63,29 @@ def make_table(results: list):
     return cds, dt
 
 
+def querying(qrystr: str = 'notanobject'):
```

Comment:
I'd recommend using some more descriptive function and variable names. You have querying, qry, Qry, qrystr all here. A longer variable name would go a long way to make the code more readable.